src/qt/optionsdialog.cpp

```diff
@@ -86,15 +86,38 @@ OptionsDialog::OptionsDialog(QWidget *parent, bool enableWallet) :
     if (!enableWallet) {
         ui->tabWidget->removeTab(ui->tabWidget->indexOf(ui->tabWallet));
     }
+	   
+	/*  Set the tooltip style here to override the OS style.*/
+    QString tooltipStyle = QString("QToolTip {color: #ffffff; background-color: #6f9bf5; font-size:11pt; border: none; border-radius:20px; padding:6px; margin:0px; min-height: 35px; qproperty-alignment: 'AlignVCenter | AlignCenter';}");
```

Comment:
Is there a reason not to add the tooltip styling to `main.css` ? Veil has too much mixing and matching of stylesheets (.ui, in the .cpp, and in main.css), it seems much more preferable to edit the main stylesheet than to do it this way. Is this specifically needed to overcome the issue or would it still work if this styling were move to main.css?

Reply:
@presstab Unfortunately the bugs in the Qt version we are using require this workaround to render the tooltips correctly. QT-QTBUG-64550 can be fixed by upgrading Qt versions. Bug QTBUG-41313 the missing background when set via CSS problem remains unresolved. 

Too make this not as bad, is there is way to make the tooltip style a global QString constant that can be reuse across the codebase?

QT-BUG: https://bugreports.qt.io/browse/QTBUG-64550
QT-BUG: https://bugreports.qt.io/browse/QTBUG-41313

Reply:
Wow that is very agitating!

Reply:
Very agitating. Hate when its a QT bug.

Reply:
Agreed! I tried every fix I could locate and the only one that worked was this. I asked in the discord dev #gui channel about upgrading the Qt version. Fuzzbawls recommended against the upgrade for a host of reasons.